client/src/public/ts/replay-control.ts

```
@@ -0,0 +1,280 @@
+ */
+function forceSetScore(score: number) {
+  variables.currentGameClientSide.shownScore = score;
+  variables.currentGameClientSide.beautifulScoreDisplayGoal = score;
+  variables.currentGameClientSide.beautifulScoreDisplayProgress = score;
+  variables.currentGameClientSide.beautifulScoreDisplayPrevious = score;
+
+  if (isNaN(score)) {
+    stageItems.textSprites.scoreText.text = "0";
+  } else {
+    stageItems.textSprites.scoreText.text =
+      parseInt(score.toString()).toLocaleString("en-US") || "0";
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)